dask_sql/physical/rex/core/call.py

```diff
@@ -943,6 +951,7 @@ class RexCallPlugin(BaseRexPlugin):
         # Temporary UDF functions that need to be moved after this POC
         "datepart": DatePartOperation(),
         "year": YearOperation(),
+        "cot": CotOperation(),
```

Comment:
The existing implementation already has cot: https://github.com/dask-contrib/dask-sql/blob/f55c27664e5a06e9405cb1372a19ca60e0c53eae/dask_sql/physical/rex/core/call.py#L914

We might be able to get by without this and just having the parser recognize this function

Reply:
You're right - it still works without the additions to `call.py`, so I'll remove them now